src/main/java/Duke.java

```diff
@@ -1,50 +1,125 @@
-                System.out.println(storedTasks[taskIndex].getStatusIcon() + " " + storedTasks[taskIndex].getDescription());
-            } else if (!command.equals("bye")) {
-                System.out.println("added: " + command);
-                storedTasks[taskNum] = new Task(command);
-                taskNum++;
+            String[] commandLine = command.split(" ", 2);
+
+            printLineBreak();
+
+            switch (commandLine[0]) {
+            case "todo":
+                addTodoTask(storedTasks, commandLine);
+                break;
+            case "deadline":
+                addDeadlineTask(storedTasks, commandLine);
+                break;
+            case "event":
+                addEventTask(storedTasks, commandLine);
+                break;
+            case "mark":
+                markTask(storedTasks, commandLine);
+                break;
+            case "unmark":
+                unmarkTask(storedTasks, commandLine);
+                break;
+            case "list":
+                listTasks(storedTasks);
+                break;
             }
 
         }
 
         System.out.println("Bye. Hope to see you again soon!");
 
     }
+
+    private static void listTasks(Task[] storedTasks) {
+        listMessage();
+        for (int i = 0; i < taskNum; i++) {
+            System.out.println((i + 1) + ". " + storedTasks[i].getTypeIcon() +
+                    storedTasks[i].getStatusIcon() + " " + storedTasks[i].getDescription());
+        }
+    }
+
+    private static void unmarkTask(Task[] storedTasks, String[] commandLine) {
+        int unmarkIndex = Integer.parseInt(commandLine[1]) - 1;
+        storedTasks[unmarkIndex].setDone(false);
+        System.out.println("OK, I've marked this task as not done yet.");
+        System.out.println(storedTasks[unmarkIndex].getTypeIcon() +
+                storedTasks[unmarkIndex].getStatusIcon() + " " + storedTasks[unmarkIndex].getDescription());
+    }
+
+    private static void markTask(Task[] storedTasks, String[] commandLine) {
+        int taskIndex = Integer.parseInt(commandLine[1]) - 1;
+        storedTasks[taskIndex].setDone(true);
+        System.out.println("Nice! I've marked this task as done.");
+        System.out.println(storedTasks[taskIndex].getTypeIcon() +
+                storedTasks[taskIndex].getStatusIcon() + " " + storedTasks[taskIndex].getDescription());
+    }
+
+    private static void addEventTask(Task[] storedTasks, String[] commandLine) {
+        String[] eventString = commandLine[1].split("/from");
+        String[] eventStartEnd = eventString[1].split("/to");
+        Event ev = new Event(eventString[0], eventStartEnd[0], eventStartEnd[1]);
+        storedTasks[taskNum] = ev;
+        addTaskMessage();
+        System.out.println("  " + ev.getTypeIcon() +
+                ev.getStatusIcon() + " " + ev.getDescription() + "(from: " + ev.getStart() + " to: " + ev.getEnd() +")");
```

Comment:
```suggestion
System.out.println("  " + ev.getTypeIcon() + ev.getStatusIcon() + " " + ev.getDescription() + "(from: " + ev.getStart() + " to: " + ev.getEnd() + ")");
```
(consistency in spaces, if all + symbols prior are sandwiched between spaces stick to that for the last one)